app/assets/stylesheets/_table.scss

```diff
@@ -0,0 +1,7 @@
+.tablesorter-header{
+  cursor: pointer;
+}
+
+.tablesorter-header:hover{
+  background-color: #efefef;
```

Comment:
Color literals like `#efefef` should only be used in variable declarations; they should be referred to via variable everywhere else.